app/views/hyrax/dashboard/collections/_list_collections.html.erb

```diff
@@ -1,6 +1,6 @@
 <% id = collection_presenter.id %>
 <tr id="document_<%= id %>">
-  <td><% unless collection_presenter.solr_document.admin_set? %><input type="checkbox" name="batch_document_ids[]" id="batch_document_<%= id %>" value="<%= id %>" class="batch_document_selector" /><% end %></td>
+  <td><input type="checkbox" name="batch_document_ids[]" id="batch_document_<%= id %>" value="<%= id %>" class="batch_document_selector" /></td>
```

Comment:
I don't understand what this change has to do with changing the message that is displayed when deleting a batch. Can you explain?